org.adempiere.base.callout/src/org/adempiere/base/callout/CalloutDocType.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * 
+ */
+package org.adempiere.base.callout;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Properties;
+
+import org.adempiere.base.IColumnCallout;
+import org.adempiere.base.annotation.Callout;
+import org.compiere.model.GridField;
+import org.compiere.model.GridTab;
+import org.compiere.model.MDocType;
+
+/**
+ * 
+ * Callout for the C_DocType table.
+ * 
+ * @author Peter Takacs, Cloudempiere
+ * 
+ */
+@Callout(tableName = MDocType.Table_Name,columnName = MDocType.COLUMNNAME_DocSubTypeSO)
+public class CalloutDocType implements IColumnCallout {
+
+private static final Map<String, IColumnCallout> calloutMap = new HashMap<String, IColumnCallout>();
+	
+	static {
+		calloutMap.put(MDocType.COLUMNNAME_DocSubTypeSO, 
+				(ctx, windowNo, mTab, mField, value, oldValue) -> isAutogenerateInOutInvoice(ctx, windowNo, mTab, mField, value));
+	}
```

Comment:
this is actually redundant since you already put in the `@Callout` annotation that this for C_DocType.DocSubTypeSO column.